pkg/kubelet/kubelet.go

```diff
@@ -705,31 +704,6 @@ func containerRef(pod *api.Pod, container *api.Container) (*api.ObjectReference,
 	return ref, nil
 }
 
-// setRef stores a reference to a pod's container, associating it with the given docker id.
```

Comment:
@vmarmol But I just moved it to manager, and use the manager as receiver for the function. Add a TODO so we can revisit this later :)